nixos/tests/zeronet-conservancy.nix

```diff
@@ -0,0 +1,24 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "zeronet-conservancy";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ fgaz ];
+  };
+
+  nodes.machine = { config, pkgs, ... }: {
+    services.zeronet = {
+      enable = true;
+      package = pkgs.zeronet-conservancy;
+    };
+  };
+
+  testScript = let
+    port = 43110;
```

Comment:
Can we get this from the service?

Reply:
no afaik, but we can do the opposite so they are in sync anyway. fixed.